src/relay/backend/te_compiler.cc

```diff
@@ -875,6 +878,121 @@ LoweredModule LowerTE(const IRModule& module, TargetMap targets, DeviceMap devic
   return lowered_module;
 }
 
+IRModule LoweredModuleToIRModule(LoweredModule mod) {
+  Map<GlobalVar, BaseFunc> unified_funcs;
+  Map<GlobalTypeVar, TypeData> unified_type_defs;
+
+  // copy main module funcs to unified funcs (what target do we need to annotate with here?)
```

Comment:
Is the `(what target do we need to annotate with here?)` a todo?

Reply:
I resolved this question, the main module functions are all Relay functions so we don't need to annotate them with the target. I will add a comment about this